pallets/common/src/lib.rs

```diff
@@ -90,14 +127,20 @@ pub mod erc;
 pub mod eth;
 pub mod weights;
 
+/// Weight info.
 pub type SelfWeightOf<T> = <T as Config>::WeightInfo;
 
+/// Collection handle contains information about collection data and id.
+/// Also provides functionality to count consumed gas.
 #[must_use = "Should call submit_logs or save, otherwise some data will be lost for evm side"]
 pub struct CollectionHandle<T: Config> {
```

Comment:
Could you add more information on how CollectionHandle is used? Something along the lines of, "CollectionHandle is used as a generic wrapper for collections of all types. It allows to perform common operations and queries on any collection type, both completely general for all, as well as their respective implementations of {link CommonCollectionOperations}". If there's anything else to make it more straightforward, feel free to add :)